src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -1420,11 +1420,17 @@ module T = struct
         ~f:(fun (seq, count) w ->
           match get_completed_work w with
           | Some cw_checked ->
-              (*If new provers can't pay the account-creation-fee then discard their work. To encourage using an existing account for snark workers*)
+              (*If new provers can't pay the account-creation-fee then discard
+              their work unless their fee is zero in which case their account
```

Comment:
Is it true that zero-balance transfers don't create the destination account? Do we have a test for this? my recollection of the sparse ledger handler is that it will still create the destination account...

Reply:
Fee transfers are not created if the prover fee is zero. I'll add a unit test

Reply:
ahh I see, happens at a different level. thanks :+1: